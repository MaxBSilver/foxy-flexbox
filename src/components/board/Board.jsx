import React from "react";
import EasyBoard from "./easy-board/EasyBoard";
import ModerateBoard from "./moderate-board/ModerateBoard";
import { useSpring, animated } from "react-spring";

export default function Board(props) {
  const springProps = useSpring({ opacity: 1, from: { opacity: 0 } });
  return props.roundNumber <= 5 ? (
    <animated.div style={springProps}>
      <EasyBoard {...props} />
    </animated.div>
  ) : (
    <animated.div style={springProps}>
      <ModerateBoard {...props} />
    </animated.div>
  );
}
