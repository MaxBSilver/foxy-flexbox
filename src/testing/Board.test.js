import React from "react";
import Board from "../components/answer/Answer";
import { shallow } from "enzyme";
describe("Board", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Board />);
  });
  it("should match the snapshot with all data passed in", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
