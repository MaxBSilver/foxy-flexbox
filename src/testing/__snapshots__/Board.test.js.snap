// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board should match the snapshot with all data passed in 1`] = `
<EasyBoard
  roundNumber={1}
/>
`;
